exec/cnex/array.c

```diff
@@ -48,3 +48,25 @@ void array_freeArray(Array *self)
     free(self);
 }
 
+Array *array_copyArray(Array *a)
+{
+    Array *r = array_createArrayFromSize(a->size);
+
+    for (size_t i = 0; i < r->size; i++) {
+        cell_copyCell(&r->data[i], &a->data[i]);
+    }
+    return r;
+}
+
+size_t array_addArrayElement(Array *a, Cell *element)
```

Comment:
Remove this if it's not used.

Reply:
Removed.  This was old code, part of MAPA().